### ISSUE TITLE
queu job error

### ISSUE THREAD

**Issue by JulieVelghe** [solspace/craft3-freeform#186]
**Describe the bug or issue you're experiencing**
when craft wants to reindex an entry (ex. it has been changed and saved) that has a form linked to it, the job in the queu fails.
It throws following error:
session_set_cookie_params(): Cannot change session cookie parameters when headers already sent
jobdata:
{
    "elementType": "Solspace\\Freeform\\Elements\\Submission",
    "elementId": 105055,
    "siteId": 1,
    "fieldHandles": [],
    "description": null
}

**Steps to reproduce**

1.
2.

**Expected behavior**

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 3.6.17
- Freeform Version: 3.11.9
- Freeform Edition:
- Fresh Install or Upgrade:

**Additional context**

**Comment by kjmartens** [comment]
Hi @JulieVelghe, sorry for the delay...

I can't personally duplicate this issue and haven't seen anyone else report this issue. I'm wondering if you might be able to provide more information about your complete setup and anything else that might be relevant here. :)

**Comment by JulieVelghe** [comment]
![screenshot-www i-mens be-2021 08 16-09_38_39](https://user-images.githubusercontent.com/46524074/129528170-1ac87f58-49f2-4bbb-8582-eb3e0f489e8b.png)
![screenshot-www i-mens be-2021 08 16-09_39_41](https://user-images.githubusercontent.com/46524074/129528275-8a47aa7f-d9b0-4a45-9768-d230eb74c237.png)
Hi, we have this issue on 2 sites now. I have attached 2 screenshots from 1 of them, showing the error in full. I've also attached the system report which shows the complete setup.
Our craft is setup in Dutch, but basically the title of the error suggests it's the standard craft indexing that fails.
I hope this information is relevant.
